ui/admin/tests/acceptance/roles/create-test.js

```diff
@@ -11,64 +11,95 @@ import {
   //invalidateSession,
 } from 'ember-simple-auth/test-support';
 
-module('Acceptance | roles', function (hooks) {
+module('Acceptance | roles | create', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
 
+  let orgURL;
+
   const instances = {
     scopes: {
       global: null,
       org: null,
       project: null,
     },
     role: null,
+    orgScope: null,
   };
   const urls = {
-    orgScope: null,
     roles: null,
     role: null,
-    newRole: null,
+    newRoleURL: null,
   };
 
   hooks.beforeEach(function () {
     authenticateSession({});
-    instances.scopes.global = this.server.create('scope', { id: 'global' });
-    instances.scopes.org = this.server.create('scope', {
-      type: 'org',
-      scope: { id: 'global', type: 'global' },
-    });
+    instances.orgScope = this.server.create(
+      'scope',
+      {
+        type: 'org',
+        scope: { id: 'global', type: 'global' },
+      },
+      'withChildren'
+    );
+
+    // instances.scopes.global = this.server.create('scope', { id: 'global' });
+    // instances.scopes.org = this.server.create('scope', {
+    //   type: 'org',
+    //   scope: { id: 'global', type: 'global' },
+    // });
     // The project scope is not yet used for role tests (though it will be
     // in the future).  This is created simply to test the grant scope loading
     // mechanism.
     instances.scopes.project = this.server.create('scope', {
       type: 'project',
-      scope: { id: instances.scopes.org.id, type: instances.scopes.org.type },
+      scope: { id: instances.orgScope.id, type: instances.orgScope.type },
     });
     instances.role = this.server.create(
       'role',
       {
-        scope: instances.scopes.org,
+        scope: instances.orgScope,
       },
       'withPrincipals'
     );
-    urls.roles = `/scopes/${instances.scopes.org.id}/roles`;
+    urls.roles = `/scopes/${instances.orgScope.id}/roles`;
     urls.role = `${urls.roles}/${instances.role.id}`;
-    urls.newRole = `${urls.roles}/new`;
+    urls.newRoleURL = `${urls.roles}/new`;
```

Comment:
Seeing as `newRole` is defined within `urls` object, `newRole` shouldn't need `URL` append to it's name.